Flaky test_parallel_replicas_over_distributed/test.py

https://s3.amazonaws.com/clickhouse-test-reports/0/966f1b6378e240652fc55c0f52ac6379a2654400/integration_tests__release__[4_4].html

```
        # parallel replicas
>       assert (
            node.query(
                f"SELECT count(), min(key), max(key), sum(key) FROM {table_name}_d",
                settings={
                    "allow_experimental_parallel_reading_from_replicas": 2,
                    "prefer_localhost_replica": prefer_localhost_replica,
                    "max_parallel_replicas": max_parallel_replicas,
                    "use_hedged_requests": 0,
                },
            )
            == expected_result
        )
E       AssertionError: assert '6002\t-1999\t1999\t2\n' == '6003\t-1999\t1999\t3\n'
E         - 6003	-1999	1999	3
E         ?    ^	     	    	^
E         + 6002	-1999	1999	2
E         ?    ^	     	    	^
```

--- a/tests/integration/test_parallel_replicas_over_distributed/test.py
+++ b/tests/integration/test_parallel_replicas_over_distributed/test.py
@@ -129,6 +129,9 @@ def test_parallel_replicas_over_distributed(
     node = nodes[0]
     expected_result = f"6003\t-1999\t1999\t3\n"
 
+    # sync all replicas to get consistent result
+    node.query(f"SYSTEM SYNC REPLICA ON CLUSTER {cluster} {table_name}")
+
     # parallel replicas
     assert (
         node.query(
@@ -143,11 +146,12 @@ def test_parallel_replicas_over_distributed(
         == expected_result
     )
 
-    # sync all replicas to get consistent result by next distributed query
-    node.query(f"SYSTEM SYNC REPLICA ON CLUSTER {cluster} {table_name}")
-
     # w/o parallel replicas
     assert (
-        node.query(f"SELECT count(), min(key), max(key), sum(key) FROM {table_name}_d")
+        node.query(f"SELECT count(), min(key), max(key), sum(key) FROM {table_name}_d",
+            settings={
+                "allow_experimental_parallel_reading_from_replicas": 0,
+           }
+        )
         == expected_result
     )

--- a/tests/integration/test_parallel_replicas_over_distributed/test.py
+++ b/tests/integration/test_parallel_replicas_over_distributed/test.py
@@ -148,10 +148,11 @@ def test_parallel_replicas_over_distributed(
 
     # w/o parallel replicas
     assert (
-        node.query(f"SELECT count(), min(key), max(key), sum(key) FROM {table_name}_d",
+        node.query(
+            f"SELECT count(), min(key), max(key), sum(key) FROM {table_name}_d",
             settings={
                 "allow_experimental_parallel_reading_from_replicas": 0,
-           }
+            },
         )
         == expected_result
     )

